lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/qm/XwikiCoreFacadeRestResource.java

```diff
@@ -92,8 +94,7 @@ public void genrationCompleted(String questionnairesId)
 		String uri = String.format("%s/learnpad/qm/corefacade/genrationcompleted/%s",
 				RestResource.REST_URI, questionnairesId);
 		PutMethod putMethod = new PutMethod(uri);
-		putMethod.addRequestHeader("Accept", "application/xml");
-
+		putMethod.addRequestHeader("Accept", MediaType.TEXT_PLAIN);
```

Comment:
@woshilapin, the operation genrationCompleted is actually a (PUT) notification to the CP saying that questionnaire whose name is in the URL is now available. The operation is not sending anything else to the CP, but the CP would probably need to query for an XML format. In conclusion, I am not sure which type to put here. But if you think XML is better, I will apply the change. Let me know.